docs/GettingStartedDocs/install_host_verify_Windows.md

```diff
@@ -0,0 +1,78 @@
+# Install the Open Enclave Host-Verify SDK NuGet Package
+
+## Platform requirements
+
+- Windows Server 2016 or 2019
```

Comment:
I just realized that this is meant for all Windows platforms. I tested it without any problem, so I updated the file so that Windows 10 is listed as a supported platform.